### DISCUSSION TITLE
How to specify the wallet address to send transactions？

### DISCUSSION THREAD

**Discussion by ovicloud** [Chia-Network/chia-blockchain#6336] [Support]
How to specify the wallet address to send transactions？
RPC interface send_transaction only contains four parameters, wallet_id，amount，address，fee，but a wallet can contain multiple addresses. How to specify a specific address to initiate a transaction。

**Comment by tiny0little** [comment]
would `-f` work?
```
$ chia wallet send -h
Usage: chia wallet send [OPTIONS]

  -f, --fingerprint INTEGER       Set the fingerprint to specify which wallet
                                  to use
```

**Comment by BrandtH22** [comment]
@ovicloud , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
